modules/administration-guide/partials/proc_configuring-minikube-github-authentication.adoc

```diff
@@ -0,0 +1,59 @@
+[id="configuring-minikube-github-authentication_{context}"]
+= Configuring Minikube with GitHub Authentication
+
+== Authentication on Kubernetes
+On {kubernetes}, link:https://github.com/oauth2-proxy/oauth2-proxy[OAuth2 Proxy] is deployed as che-gateway sidecar ensuring requests authentication. Then {kubernetes} must have configured OIDC issuer to know about users identities and CheCluster CR must be configured with:
```

Comment:
should we somehow explicitly says that OIDC is a requirement now for using che with k8s.
Maybe add a reference https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens

Reply:
maybe a separate page?

Reply:
see https://github.com/eclipse/che-docs/pull/2187/files#diff-5f3256312ccf179e558b8ce382575d79f96be22c8982ef26e1893f4151200e26R42
I haven't found any better place and @l0rd has created this issue https://github.com/eclipse/che/issues/20909 to provide help how to setup OIDC on various Kubernetes clusters. However, it will need some deeper investigation.